packages/extensions/openapi-to-typespec/src/utils/model-generation.ts

```diff
@@ -14,7 +18,12 @@ export function getModelPropertiesDeclarations(properties: TypespecObjectPropert
     if (property.defaultValue) {
       defaultValue = ` = ${property.defaultValue}`;
     }
-    definitions.push(`"${property.name}"${getOptionalOperator(property)}: ${property.type}${defaultValue};`);
+    if (isFullCompatible && property.suppressions) {
```

Comment:
why we need `isFullCompatible` here? it is for different generation for brownfield and greenfield.

Reply:
My understanding is greenfield should have no suppressions because they don't need backward compatibility. Correct me if I'm wrong.

Reply:
Synced offline, keep this logic until we need to change.